origin-growth/src/resources/invite.js

```diff
@@ -55,6 +55,50 @@ class GrowthInvite {
     })
   }
 
+  /**
+   * Creates a referrer - referee connection in case this referee does not have a referrer yet
+   *
+   * @param {string} code - growth invitation code
+   */
+  static async makeReferralConnection(code, walletAddress) {
+    try {
+      const referralLink = await db.GrowthReferral.findOne({
+        where: {
+          refereeEthAddress: walletAddress
+        }
+      })
+      const referrer = await GrowthInvite._getReferrer(code)
+
+      console.log("Referral link: ", JSON.stringify(referralLink))
+      if (referralLink && referralLink.referrerEthAddress.toLowerCase() !== referrer.toLowerCase()) {
+        /* The referrer present in the referee's identity does not match
```

Comment:
nit: we decided to no longer store invite code in referrer's identity so update the comment.
maybe something like: "the referrer associated with the invite code does not match..."

Reply:
👍